config/ceh-production.yml

```diff
@@ -5,7 +5,7 @@ datalabName: datalab
 domain: datalabs.ceh.ac.uk
 kubernetesMasterHostName: ceh-datalabs-k8s-master-1
 
-datalabVersion: 0.24.0-48-g2ae9e70
+datalabVersion: 0.28.0-17-g1c79812 
```

Comment:
Latest tag now is 0.28.0-21-gd7dfc23 which I have tested if this could be deployed instead